_rules/unique-attrs-e6952f.md

```diff
@@ -15,6 +15,10 @@ input_aspects:
 authors:
   - Wilco Fiers
   - Emma Pratt Richens
+htmlHintIgnore:
```

Comment:
What is this, and is this something we will have to add to rules ourselves going forward?

Reply:
As explained over a call, we do not have to do this for all the rules. Only for rules where a lint rule needs to be overridden.

Reply:
Additionally, this is *not* something that rule writers or PR reviewers will have to care about. Not having it where it's needed will *not* hamper the full review process. It will prevent merging of the PR, but it does not impact the (lengthy) discussion that happens before, nor will it prevent anybody from accepting the PR or requesting changes.

And it is ultimately a minor technical point where anybody can ask help from people with a developer background to investigate or fix it.